x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -546,7 +546,7 @@ export const INITIAL_PROMPT_TEXT = i18n.translate(
 export const OVERWRITE_WITH_SAME_NAME = i18n.translate(
   'xpack.securitySolution.detectionEngine.components.importRuleModal.overwriteDescription',
   {
-    defaultMessage: 'Automatically overwrite saved objects with the same rule ID',
+    defaultMessage: 'Overwrite existing Rules with conflicting Rule ID',
```

Comment:
Sorry for not spotting this earlier, but not sure if `Rule(s)` should be capitalized here? It's not capitalized above, so at the very least we should be consistent. Also not sure on wording here since I believe the docs call them `Detection Rules` instead of `Security rules`. @elastic/security-docs, would you be able to chime in here?

Reply:
Hi @spong @dplumlee. In terms of syntax, `rules` should be lowercase. With the new nav, however, I'm unclear if they should still be called `detection rules` or just `rules` now. I'm OK with `prebuilt rules` in this context, but wasn't sure if the D&R team came to a consensus about it.

Reply:
I think we should go with `detection rules` in this context as it doesn't just affect what we usually refer to as `prebuilt` rules. If there aren't any other issues, I'll go with the change pushed up [most recently](https://github.com/elastic/kibana/pull/106188/commits/836d8766291ff728bba534ef2b5d5f77b8a4e9e7)